subiquity/controllers/ceph.py

```diff
@@ -23,6 +23,13 @@
 
 
 class CephDiskController(BaseController):
+    signals = [
```

Comment:
Possibly, this reflects the API of urwid.connect_signal more closely, which is what all this signal stuff backs onto. I may attempt to delete all the signal stuff in favour of calling methods on objects fairly soon...